packages/@ama-sdk/core/schematics/ng-add/index.ts

```diff
@@ -7,5 +9,27 @@ import type { Rule } from '@angular-devkit/schematics';
  */
 export function ngAdd(): Rule {
   /* ng add rules */
-  return () => {};
+  return (tree: Tree) => {
```

Comment:
Here it would be good to use the technique of dynamic import to be able to display a custom message if the '@o3r/schematics' is not yet installed.
here is an example of how we did for [eslint-config-otter package](https://github.com/AmadeusITGroup/otter/pull/19/files#diff-5a4d821d2e217ee51ec518db673d86d72831b1c743116f5f7b00f10bac755fff)